### PR TITLE
Intake band updates

### PR DESCRIPTION
**Pull request by MarcoAlejandroLopezGomez**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import com.arcrobotics.ftclib.command.CommandBase;
+
+import org.firstinspires.ftc.teamcode.Subsystems.Shooter;
+
+
+public class MoveShooter extends CommandBase {
+    private Shooter shooter;
+    private double voltage;
+
+    public MoveShooter(Shooter subsystem, double voltage) {
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java]: Not a voltage

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        this.voltage = voltage;
+        shooter = subsystem;
+        addRequirements(subsystem);
+    }
+
+    @Override
+    public void initialize(){
+        shooter.Voltage(voltage);
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java]: Not a voltage

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java [hunk 3]
```diff
@@ -1,109 +1,77 @@
...
 
     @Override
     public void runOpMode() {
-        driverGamepad = gamepad1;
-        operatorGamepad = gamepad2;
-
         chassis     = new Chassis(hardwareMap);     // Create an instance of Chassis
         elevator    = new Elevator(hardwareMap);    // Create an instance of Elevator
-        intake      = new Intake(hardwareMap);      // Create an instance of Intake
+        intake      = new Intake(hardwareMap, "intake");      // Create an instance of Intake
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java]: Inside the Intake Subsystem

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Shooter.java [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+public class Shooter extends SubsystemBase {
     // Motor Declaration
     private Servo Shoot_Servo;
 
     public Shooter (HardwareMap hardwareMap){
         Shoot_Servo = hardwareMap.get(Servo.class, "shoot_Servo");
     }
 
-    public void ShooterVoltage(double ShooterMotorPosition) {
+    public void Voltage(double ShooterMotorPosition) {
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Shooter.java]: Change

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
 
-public class Intake {
-    // Motor Declaration
     private DcMotor intake;
 
-    public Intake(HardwareMap hardwareMap) {
-        intake = hardwareMap.get(DcMotor.class, "intake");
-    }
+        public Intake(final HardwareMap hMap, final String IntakeSubsystem) {
+            intake = hMap.get(DcMotor.class, IntakeSubsystem);
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java (hunk 1 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java (hunk 2 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/MoveShooter.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java (hunk 3 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MainSystem.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Shooter.java (hunk 4 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Shooter.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java (hunk 5 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java]: Fix

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 
-public class Intake {
-    // Motor Declaration
     private DcMotor intake;
 
-    public Intake(HardwareMap hardwareMap) {
-        intake = hardwareMap.get(DcMotor.class, "intake");
-    }
+        public Intake(HardwareMap hardwareMap) {
+            intake = hardwareMap.get(DcMotor.class, "IntakeSubsystem");
```
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java]: Not the name of the motor

**Review by SantiQ0905 — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java (hunk 6 above)
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Subsystems/Intake.java]: Fix

**Review by SantiQ0905 — APPROVED** [review]

**Pull request merged**